src/adapter/src/coord/command_handler.rs

```diff
@@ -109,13 +111,8 @@ impl Coordinator {
             }
 
             Command::DumpCatalog { session, tx } => {
-                // TODO(benesch/jkosh44): when we have RBAC, dumping the catalog should
```

Comment:
I think this comment needs to stay.

Reply:
I'm pretty sure we're handling it [here](https://github.com/MaterializeInc/materialize/blob/c8ceec0ab8d015eb93e483f0c746cfda711a0894/src/adapter/src/rbac.rs#L122-L130), more than happy to re-add it if I'm wrong though!